SHARE-435 Studios details view

mahmoudZkhalil opened the pull request:
FE studio details page
add/list/remove comments and replies
list/remove projects
update studio cover
php unittest

---
### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catroweb-Symfony/blob/develop/.github/contributing.md) and [wiki pages](https://github.com/Catrobat/catroweb-Symfony/wiki/) of this repository.

- [x] Include the name and id of the Jira ticket in the PR’s title eg.: `SHARE-666 The devils ticket`
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no warnings and errors 
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Verify that all tests are passing (CI), if not please state the test cases in the [section](#Tests) below
- [x] Perform a self-review of the changes
- [x] Stick to the project’s git workflow (rebase and squash your commits)
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] Put your ticket into the `Code Review` section in [Jira](https://jira.catrob.at/)
- [x] Post a message in the *#catroweb* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer
- [x] Check that your pull request has been successfully deployed to https://web-test-1.catrob.at/

### Additional Description
`TODO: Add additional information that is not in your commit-message here`

### Tests - additional information
`TODO: add additional information about testruns here`

HCrane commented on migrations/2021/Version20210530193145.php:
```diff
@@ -20,10 +20,10 @@ public function getDescription(): string
+        $this->addSql('CREATE TABLE studio (id CHAR(36) NOT NULL COMMENT \'(DC2Type:guid)\', name VARCHAR(255) NOT NULL, description TEXT NOT NULL, is_public TINYINT(1) DEFAULT \'1\' NOT NULL, is_enabled TINYINT(1) DEFAULT \'1\' NOT NULL, allow_comments TINYINT(1) DEFAULT \'1\' NOT NULL, cover_path VARCHAR(300) DEFAULT NULL, updated_on DATETIME DEFAULT NULL, created_on DATETIME NOT NULL, UNIQUE I ...
+        $this->addSql('CREATE TABLE studio_activity (id INT AUTO_INCREMENT NOT NULL, studio CHAR(36) NOT NULL COMMENT \'(DC2Type:guid)\', user CHAR(36) NOT NULL COMMENT \'(DC2Type:guid)\', type ENUM(\'comment\', \'project\', \'user\'), created_on DATETIME NOT NULL, INDEX IDX_D076B8584A2B07B6 (studio), INDEX IDX_D076B8588D93D649 (user), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8m ...
+        $this->addSql('CREATE TABLE studio_program (id INT AUTO_INCREMENT NOT NULL, studio CHAR(36) NOT NULL COMMENT \'(DC2Type:guid)\', activity INT NOT NULL, program CHAR(36) NOT NULL COMMENT \'(DC2Type:guid)\', user CHAR(36) NOT NULL COMMENT \'(DC2Type:guid)\', updated_on DATETIME DEFAULT NULL, created_on DATETIME NOT NULL, INDEX IDX_4CB3C24A4A2B07B6 (studio), UNIQUE INDEX UNIQ_4CB3C24AAC74095 ...
+        $this->addSql('CREATE TABLE studio_user (id INT AUTO_INCREMENT NOT NULL, studio CHAR(36) NOT NULL COMMENT \'(DC2Type:guid)\', activity INT NOT NULL, user CHAR(36) NOT NULL COMMENT \'(DC2Type:guid)\', role ENUM(\'admin\', \'member\'), status ENUM(\'active\', \'banned\', \'pending_request\'), updated_on DATETIME DEFAULT NULL, created_on DATETIME NOT NULL, INDEX IDX_EC686DD14A2B07B6 (studio) ...
```
Needs checking if allowed with production server!

HCrane commented on translations/catroweb.en.yml:
```diff
@@ -779,4 +779,27 @@ tags:
+    projects: 'projects'
+    comments: 'comments'
+    public: 'public'
+    private: 'private'
```
Check capitalization

mahmoudZkhalil replied:
will be capitalized through css for 'projects' and 'comments' and 'public' and 'private' have small initials

HCrane commented on translations/catroweb.en.yml:
```diff
@@ -779,4 +779,27 @@ tags:
+    join: 'Join'
+    leave: 'Leave'
+    showMore: 'show more'
+    showLess: 'show less'
```
Check capitalization

HCrane replied:
Already exits. Please use those.

HCrane reviewed (dismissed):
Approved when the migration stuff is fixed.

mahmoudZkhalil committed:
SHARE-435 Studios details view

FE studio details page
add/list/remove comments and replies
list/remove projects
update studio cover
php unittest
